PhysicsTools/NanoAOD/python/triggerObjects_cff.py

```diff
@@ -27,44 +58,43 @@
             l1seed = cms.string("type(-98)"),  l1deltaR = cms.double(0.3),
             #l2seed = cms.string("type(92) && coll('')"),  l2deltaR = cms.double(0.5),
             skipObjectsNotPassingQualityBits = cms.bool(True),
-            qualityBits = cms.string(
-                            "filter('*CaloIdLTrackIdLIsoVL*TrackIso*Filter') + " \
-                            "2*filter('hltEle*WPTight*TrackIsoFilter*') + " \
-                            "4*filter('hltEle*WPLoose*TrackIsoFilter') + " \
-                            "8*filter('*OverlapFilter*IsoEle*PFTau*') + " \
-                            "16*filter('hltEle*Ele*CaloIdLTrackIdLIsoVL*Filter') + " \
-                            "32*filter('hltMu*TrkIsoVVL*Ele*CaloIdLTrackIdLIsoVL*Filter*')  + " \
-                            "64*filter('hlt*OverlapFilterIsoEle*PFTau*') + " \
-                            "128*filter('hltEle*Ele*Ele*CaloIdLTrackIdLDphiLeg*Filter') + " \
-                            "256*max(filter('hltL3fL1Mu*DoubleEG*Filtered*'),filter('hltMu*DiEle*CaloIdLTrackIdLElectronleg*Filter')) + " \
-                            "512*max(filter('hltL3fL1DoubleMu*EG*Filter*'),filter('hltDiMu*Ele*CaloIdLTrackIdLElectronleg*Filter')) + " \
```

Comment:
This line was skipped below, which should explain the unexpected changes in the electron objects

Reply:
thanks for spotting